src/examples/custom.rs

```diff
@@ -9,10 +8,11 @@ use crate::{
         CustomPredicateBatch, CustomPredicateRef, NativePredicate as NP, Params, PodType,
         Predicate, KEY_SIGNER, KEY_TYPE,
     },
+    Error,
```

Comment:
Can we use `Result` here?  Then we don't need to explicitly say `Result<..., Error>` and just do `Result<...>`